EIPS/eip-1.md

```diff
@@ -315,12 +315,12 @@ Permitted W3C recommendation URLs MUST anchor to a specification in the technica
 Links to WHATWG specifications may be included using normal markdown syntax, such as:
 
 ```markdown
-[HTML](https://html.spec.whatwg.org/commit-snapshots/578def68a9735a1e36610a6789245ddfc13d24e0/)
+[HTML](https://html.spec.whatwg.org/commit-snapshots/578def68a9735a1e36610a6789245ddfc13d24e0)
 ```
 
 Which renders as:
 
-[HTML](https://html.spec.whatwg.org/commit-snapshots/578def68a9735a1e36610a6789245ddfc13d24e0/)
+[HTML](https://html.spec.whatwg.org/commit-snapshots/578def68a9735a1e36610a6789245ddfc13d24e0)
```

Comment:
The canonical location of this document has the trailing slash. Why remove it? Similarly, why remove it everywhere else?

Reply:
The idea was to being consistent to all URLs. (Either all with trailing slash, or all without trailing slash.)

Because trailing slashes do not (unless I'm mistaken) bring any information, I chose to remove them all.